### PR TITLE
fix(create): bad grammar in template docs

### PR DESCRIPTION
**Pull request by naffydharni006**
Corrected grammar

## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).

## Motivation

It didn't look good that such a nice tool has a grammar mistake in their preset 😅

## Test Plan

Corrected grammar in translation docs page of classic-typescript preset.

### Test links

Deploy preview: https://deploy-preview-7978--docusaurus-2.netlify.app/

## Related issues/PRs

None

### CODE REVIEW

File: examples/classic-typescript/docs/tutorial-extras/translate-your-site.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Your localized site is accessible at [http://localhost:3000/fr/](http://localhos
 
 :::caution
 
-In development, you can only use one locale at a same time.
+In development, you can only use one locale at the same time.
```
Comment (slorber) [inline examples/classic-typescript/docs/tutorial-extras/translate-your-site.md:46]: Thanks

Examples shouldn't be updated, they are generated from our init templates

The fix should be done here instead: packages/create-docusaurus/templates/shared/docs/tutorial-extras/translate-your-site.md

**Review by slorber — CHANGES_REQUESTED** [review]

File: examples/classic-typescript/docs/tutorial-extras/translate-your-site.md (hunk 1 above)
Comment (Josh-Cena) [inline examples/classic-typescript/docs/tutorial-extras/translate-your-site.md:46]:
```suggestion
In development, you can only use one locale at a time.
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

File: examples/classic-typescript/docs/tutorial-extras/translate-your-site.md (hunk 1 above)
Comment (naffydharni006) [reply examples/classic-typescript/docs/tutorial-extras/translate-your-site.md]: Updated the `create-docusaurus` package. Should the commit changing examples be reverted?

File: examples/classic-typescript/docs/tutorial-extras/translate-your-site.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Your localized site is accessible at [http://localhost:3000/fr/](http://localhos
 
 :::caution
 
-In development, you can only use one locale at a same time.
+In development, you can only use one locale at a time.
```
Comment (Josh-Cena) [inline examples/classic-typescript/docs/tutorial-extras/translate-your-site.md:46]:
```suggestion
In development, you can only use one locale at a same time.
```

**Review by Josh-Cena — APPROVED** [review]
Thanks!

**Pull request merged**